pom.xml

```diff
@@ -91,8 +91,8 @@
         <artifactId>maven-compiler-plugin</artifactId>
         <version>3.1</version>
         <configuration>
-          <source>1.6</source>
-          <target>1.6</target>
+          <source>1.8</source>
```

Comment:
Should we change the target JVM version in this PR? If not, remove this change.

Reply:
Compiling did not work for me under 1.6. If that’s not needed it can be removed

Reply:
Understood. Anyway I have no intention to support below 1.8, we can make 1.8 as the default targe.t